### PR TITLE
feat(cargo-shuttle): shuttle binary

### PR DESCRIPTION
**Pull request by jonaro00**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -167,6 +167,7 @@ commands:
           command: |
             mkdir $BIN_DIR
             mv target/<< parameters.target >>/release/cargo-shuttle<< parameters.suffix >> $BIN_DIR/cargo-shuttle<< parameters.suffix >>
+            mv target/<< parameters.target >>/release/shuttle<< parameters.suffix >> $BIN_DIR/shuttle<< parameters.suffix >>
```
Comment (jonaro00) [inline .circleci/config.yml:170]: The corresponding change in install.sh will be in a separate PR and shall be merged after the artifacts of the next release are published.

**Review by oddgrd — APPROVED** [review]

**Comment by chenrui333** [comment]
@jonaro00 @oddgrd 👋 looks like with this patch, now you have binaries doing the same? Thanks!

**Comment by jonaro00** [comment]
@chenrui333 The new `shuttle` binary is for using the new shuttle.dev platform, with its updated commands and API.

**Comment by chenrui333** [comment]
got you make sense, this is helpful! 💯 

relates to https://github.com/Homebrew/homebrew-core/pull/201877
